docs/execution_flows.md

```diff
@@ -35,19 +35,19 @@ title: Safe{Core} Protocol High-level Plugin execution flow
 ---
 flowchart TD;
    TxExecuteFromPlugin(Call to Plugin to execute tx ) --> ExamplePlugin1
-   Safe --> Execute_Transaction_From_Plugin(Execute transaction)
-   Validate_ExecuteFromPluginFlow -- Yes --> Safe(Safe Account)
+   Account --> Execute_Transaction_From_Plugin(Execute transaction)
+   Validate_ExecuteFromPluginFlow -- Yes --> Account(Safe{Core} Protocol Account)
```

Comment:
Error is because of this line. Should be wrapped in quotes if special character is used

```
Validate_ExecuteFromPluginFlow -- Yes --> Account("Safe{Core} Protocol Account")
```

Reply:
Done https://github.com/safe-global/safe-core-protocol/pull/100/commits/3545679bcc51718ab3e7ec71bf73fb55c38f74b4